README.md

```diff
@@ -188,11 +188,6 @@ Please refer [CurlHTTPClient](/src/LINEBot/HTTPClient/CurlHTTPClient.php) that i
 See also
 --
 
-### [line-bot-sdk-tiny](./line-bot-sdk-tiny)
```

Comment:
Note: In 2023, PHP developers can use composer. There's no reason to provide a simple version of the SDK.